webpack.config.js

```diff
@@ -99,7 +118,11 @@ function html_pug_plug(title, template, opts) {
 }
 
 module.exports.plugins = [
-    html_pug_plug("A Tour of C++", "templates/index.pug"),
+    //Inject lessons index through options.
```

Comment:
Could we agree on having a space between `//` and the first letter, to stay consistent in our comment style?

Reply:
Ok